lumicks/pylake/detail/widefield.py

```diff
@@ -533,11 +532,17 @@ def rotation(cls, theta, center):
         Parameters
         ----------
         theta: float
-            angle of rotation
+            angle of rotation in degrees (counter-clockwise)
         center: np.ndarray
             (x, y) point, center of rotation
         """
-        return cls(cv2.getRotationMatrix2D(tuple(center), theta, scale=1.0))
+        from skimage.transform import SimilarityTransform
```

Comment:
Out of interest: Is there a specific reason to no use [`EuclidianTransform`](https://scikit-image.org/docs/stable/api/skimage.transform.html#skimage.transform.EuclideanTransform)?

Reply:
Good point. We're not using scaling, so let's switch it over to `Euclidian`.